_generated/MavenAuthenticateV0/mavenauth.ts

```diff
@@ -54,6 +60,33 @@ async function run(): Promise<void> {
             tl.debug(tl.loc("Info_CreatingSettingsXml", userSettingsXmlPath));
             tl.setVariable('FIRST_RUN_SETTINGS_XML_EXISTS_PATH', userSettingsXmlPath);
         }
+        
+        if (feedUrl && entraWifServiceConnectionName) {
+            
+            tl.debug(tl.loc("Info_AddingFederatedFeedAuth", entraWifServiceConnectionName, feedUrl));
+            const feedTenant = await getFeedTenantId(feedUrl);
+            let token = await getFederatedWorkloadIdentityCredentials(entraWifServiceConnectionName, feedTenant);
+            
+            if (token) {
+
+                const wifServerElement = {
+                    id: entraWifServiceConnectionName,
+                    username: 'WIFbuild',
```

Comment:
Consider using the service connection name here instead.

Reply:
Ok, changed this!

Reply:
Hey, the more I think about it, I'm not sure if this is the best idea. It seems like the entraWifServiceConnectionName input gets converted into a guid, so it might be confusing to just have this as both the id and username. If users want to know what specific connection was authenticated they can still look at the id, but maybe we want the username to indicate that it's for an Entra Workload ID-Backed Azure DevOps User service connection?

Reply:
I ended up changing this back in my latest commit because I thought the original would be less confusing to users